# Hardware info string

uint64 timestamp		# time since system start (microseconds)
char[64] hw_info		# main autopilot hw info string
